base/install-packages

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -o errexit
+
+if [ $# = 0 ]; then
+  echo >&2 "No packages specified"
+  exit 1
+fi
+
+DEBIAN_FRONTEND=noninteractive
+
+DAZZLE_MARKS="/var/lib/apt/dazzle-marks/"
+TIMESTAMP=$(date +%s)
+
+if [ ! -d "${DAZZLE_MARKS}" ]; then
+  mkdir -p "${DAZZLE_MARKS}"
+fi
+
+apt-get update
+apt-get install -yq --no-install-recommends $@
```

Comment:
Adding `--no-install-recommends` will save some space, but also means that many packages that were previously installed as a side effect will now be missing.

This might inadvertently break downstream images / project setups that might have relied on these packages to be present.

While we could arguably consider this an "end-user problem", I think we should extensively test the images produced by this change, for example against the 5 most common projects on Gitpod.

Reply:
@jankeromnes Could this possibly be inadvertently causing this [issue](https://github.com/apolopena/gitpod-laravel8-starter/issues/46) I am having in all of my projects? Or [this line](https://github.com/gitpod-io/workspace-images/commit/6a8181b1c37f99b68aed2046abbe7c6381249187#r47850267) of code maybe?
I also posted the problem I am having with all my projects in the gitpod community forum [here](https://community.gitpod.io/t/var-log-apache2-not-found-by-default/2876). I don't mean to convolute or hijack this great PR. I just wanted to a stab at finding the root cause.
Thanks.